src/jammed.js

```diff
@@ -401,6 +385,141 @@ Method.prototype.draw = function(x, y) {
   }
   text += ")";
   apo.ctx.fillText(text, x+10+len.width, y);
+};console.log('loaded samplejs');
```

Comment:
This should not be "jammed" into jammed.js, but instead loaded into the html after jammed.js is loaded.

These functions bellow until line 433 are the .js the html will import after apoena to instanciate its canvas and elements.